packages/@aws-cdk/aws-lambda-python/lib/function.ts

```diff
@@ -0,0 +1,71 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import { Bundling } from './bundling';
+
+/**
+ * Properties for a PythonFunction
+ */
+export interface PythonFunctionProps extends lambda.FunctionOptions {
+  /**
+   * Path to the (Python) entry file.
+   */
+  readonly entry: string;
```

Comment:
I wonder if it makes more sense that this would be the root directory (where you'd look for `requirements.txt`) and add another (optional) option for the index file? (defaults to `index.py`).

The current situation precludes the option of putting your entry point in a subdirectory...